src/javascript/app_2/Stores/Modules/Trading/Helpers/proposal.js

```diff
@@ -39,7 +39,7 @@ const createProposalRequestForContract = (store, type_of_contract) => {
 
     if (store.expiry_type === 'endtime') {
         const server_time = store.root_store.common.server_time;
-        const start_date  = moment.unix(store.start_date || (server_time / 1000)).utc();
+        const start_date  = moment.unix(store.start_date || (server_time / 1000).toFixed(0)).utc();
```

Comment:
since `store.root_store.common.server_time` is a moment object it doesn't need to divide it by 1000.